test/Microsoft.AspNetCore.Sockets.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -199,6 +200,52 @@ public HttpConnectionDispatcherTests(ITestOutputHelper output) : base(output)
             }
         }
 
+        [InlineData(TransportType.LongPolling)]
+        [InlineData(TransportType.ServerSentEvents)]
+        [Theory]
+        public async Task PostSendsToConnection(TransportType transportType)
+        {
+            using (StartLog(out var loggerFactory, LogLevel.Debug))
```

Comment:
Do something like `$"{nameof(PostSendsToConnection}_{transportType}"` for the test name

Reply:
Other tests don't do this.